vizro-core/tests/unit/vizro/models/_controls/test_parameter.py

```diff
@@ -70,14 +70,32 @@ class TestPreBuildMethod:
             ),
         ],
     )
-    def test_target_and_title_generation_valid(self, test_input, title):
+    def test_set_target_and_title_valid(self, test_input, title):
         parameter = Parameter(targets=["scatter_chart.x"], selector=test_input)
         page = model_manager["test_page"]
         page.controls = [parameter]
         parameter.pre_build()
         assert parameter.targets == ["scatter_chart.x"]
         assert parameter.selector.title == title
 
+    @pytest.mark.parametrize(
+        "test_input",
+        [
+            (vm.Checklist(options=["lifeExp", "gdpPercap", "pop"], value=["lifeExp"])),
+            (vm.Dropdown(options=["lifeExp", "gdpPercap", "pop"], multi=False, value="lifeExp")),
+            (vm.RadioItems(options=["lifeExp", "gdpPercap", "pop"], value="lifeExp")),
+        ],
+    )
+    def test_set_actions_valid(self, test_input):
```

Comment:
Could we to round this of once and for all also split the parameter pre-build into functions, align the naming with filtes, and have the test refer to these splits.

I think if we do it now, we don't have to touch these again, which would be quite nice.